scripts/build_images/build.py

```
@@ -202,6 +203,16 @@ def build_image(
     print(f"Metadata: {metadata.strip()}")
     digest = json.loads(metadata)["containerimage.digest"]
 
+    # Query local registry for image size.
+    manifest_url = f"http://localhost:5000/v2/{image}/manifests/{digest}"
+    req = urllib.request.Request(
+        manifest_url,
+        headers={"Accept": "application/vnd.docker.distribution.manifest.v2+json"},
+    )
```

Ruff (0.15.0)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)